docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-overview-con.adoc

```diff
@@ -0,0 +1,41 @@
+[id='case-management-overview-con']
+= Case management
+
+Case management is an extension of Business Process Management (BPM) that allows you to handle dynamic and ad hoc activities. 
+
+BPM is a management practice for automating tasks that are repeatable and have a common pattern, with a focus on optimization by perfecting a process. Business processes are usually modeled with clearly defined paths leading to a business goal, often with a starting point that is structurally connected to build an end to end flow of work and data. This requires a lot of predictability, usually based on mass-production principles. However, many applications in the real world cannot be described completely from start to finish (including all possible paths, deviations, and exceptions). Moreover, using a process oriented approach in certain cases may lead to complex solutions that are hard to maintain. 
+
+Case management provides problem resolution for non-repeatable, unpredictable process instances as opposed to the efficiency oriented approach of Business Process Management for routine predictable tasks. It handles one-off situations where the process cannot be predicted in advance. Case definition usually consists of loosely coupled process fragments that can be connected directly or indirectly to lead to certain milestones and ultimately, a business goal, while the process is managed dynamically in response to changes that occur during runtime. 
+
+Case management solutions are template-based, with users having control over processes that can change on a case-by-case basis. As work cannot be predefined in case management, each case instance must have a goal, a deadline, and a predefined product or outcome. Case management is for such collaborative and dynamic tasks that require human actions, therefore a person needs to own the case and manage the process, making the necessary changes to reach a case resolution.
+
+In {PRODUCT}, case management includes the following core engine features:
+
+* case file instance
+* per case session strategy
+* case comments
+* milestones
+* stages
+* ad hoc fragments
+* dynamic tasks
+* case identifier (correlation key)
+* case lifecycle (close, reopen, resolve, etc)
```

Comment:
there is no such life cycle phase like resolve - it's close, reopen, cancel and destroy